utils/converter.go

```diff
@@ -31,6 +32,23 @@ func ConvertMapToString(mapString map[string]string) (string, error) {
 	return string(stringPractitionerLinks), nil
 }
 
+// ConvertStringToArray is a helper function to split string to array string
+func ConvertStringToArray(stringItem string, splitChar string) []string {
+	return strings.Split(stringItem, splitChar)
+}
+
+// CheckStringArrayHasElement is a helper function to check if an element is in string array
+func CheckStringArrayHasElement(stringItem string, splitChar string, find string) bool {
```

Comment:
Nitpicking: this (now) takes a string not a string array so a better name might be CheckStringContainsElement?